l10n_ch_base_bank/tests/test_bank.py

```diff
@@ -281,69 +284,3 @@ def test_name_search(self):
         self.bank.city = 'Lausanne-Centre'
         result = self.env['res.bank'].name_search('Lausanne-Centre')
         self.assertEqual(result and result[0][0], self.bank.id)
-
-    def test_multiple_isr_bank_account_for_same_partner(self):
```

Comment:
We want to keep a similar test to check the constraint will not forbid us to do that.

Your company can have 2 accounts at the same bank for different purposes. Thus we want to be sure we can have twice the same ccp.

By the way we might be missing a test having 2 partners with same bank ccp.